schema.sql

```diff
@@ -1,4 +1,4 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals ( id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY, name TEXT NOT NULL, date_of_birth DATE NOT NULL, escape_attempts INT NOT NULL, neutered BOOLEAN NOT NULL,weight_kg DECIMAL NOT NULL );
+CREATE TABLE animals ( id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY, name TEXT NOT NULL, date_of_birth DATE NOT NULL, escape_attempts INT NOT NULL, neutered BOOLEAN NOT NULL,weight_kg DECIMAL NOT NULL, species TEXT );
```

Comment:
- [ ] Instead of editing the previous animal schema directly, you should have just had another query to alter the animal table. Something like the example below.

`ALTER TABLE animals ADD species varchar(255);`